lms/templates/emails/html/activation_email.html

```diff
@@ -2,6 +2,7 @@
 <%!
   from microsite_configuration.middleware import microsite
   from django.utils.translation import ugettext as _
+  from django.conf import settings
```

Comment:
In this template there are calls to settings module without importing it so I think this import is not necessary. Anyways please test it

Reply:
I agree with you here but I'm not sure how to test it. Since this is the content of the activation email I'd need to send an email to myself from the my local environment ... I know you can do that in django but it may take me some time to figure out how :/